config.inc.php.dist

```diff
@@ -6,12 +6,12 @@ define('USER', 'guest');
 define('PASS', 'guest');
 define('VHOST', '');
```

Comment:
Same think (see [below](https://github.com/CanalTP/AmqpMttWorkers/pull/2/files#diff-e83d47e0e113314846de23f7425eb13cL10)). You can add 'localhost'.